etc/argus.conf

```diff
@@ -25,3 +25,10 @@ require_sysprep = True
 
 created_user = Admin
 group = Administrators
+
+[debug]
```

Comment:
You could modify the argus.conf.sample file to provide more info regarding the [debug] mode and it's fields.